sdk/include/cardboard.h

```diff
@@ -383,21 +391,28 @@ void CardboardHeadTracker_resume(CardboardHeadTracker* head_tracker);
 
 /// Gets the predicted head pose for a given timestamp.
 ///
+/// @details  @p viewport_orientation will incur in an extra rotation to the head rotation to
+///           align the pose with the viewport orientation. 
+///           CardboardViewportOrientation_LandscapeLeft introduces an identity rotation to the
+///           head tracker pose.
+///
 /// @pre @p head_tracker Must not be null.
 /// @pre @p position Must not be null.
 /// @pre @p orientation Must not be null.
+/// @pre @p viewport_orientation Must be a valid CardboardViewportOrientation.
```

Comment:
Consider validating the enum value in `cardboard.cc`. In case it is not valid, please add a log message.

Reply:
Considering that the project does not validate the other carboard.h enum (`CardboardEye`). Let me show an example: 
https://github.com/googlevr/cardboard/blob/master/sdk/cardboard.cc#L143

I followed the same criteria I did not validate this either. I also had to change the documentation for this.

Let me know if that is right or I should still validate this enum.